packages/angular_devkit/core/src/json/schema/registry.ts

```diff
@@ -624,23 +625,44 @@ export class CoreSchemaRegistry implements SchemaRegistry {
 
     return from(provider(prompts)).pipe(
       map(answers => {
-        for (const path in answers) {
-          const pathFragments = path.split('/').map(pf => {
-            if (/^\d+$/.test(pf)) {
-              return pf;
-            } else {
-              return '\'' + pf + '\'';
-            }
-          });
+        const pathFragmenter = (pf: string) => {
```

Comment:
I don't think these changes are needed.  `answers` should always be an object.

Reply:
You are right! I guess when I was first working on this the answers were coming through as an array of the selected values. I realize now that was an artifact of my erroneous test case. I will make the change right away! For making changes, what is the standard for the commits? @clydin

Reply:
The process is generally to make additional commits locally; then squash (`git rebase -i master`) the fix-up commits into the relevant previously existing commit.